lms/djangoapps/bookmarks/tests/test_views.py

```diff
@@ -199,8 +229,17 @@ def test_get_bookmarks_with_invalid_data(self):
         response = self.send_get(client=self.client, url=reverse('bookmarks'), query_parameters='course_id=invalid')
         bookmarks = response.data['results']
         self.assertEqual(len(bookmarks), 0)
+        self.assert_bookmark_listed_event_emitted(
+            mock_tracker,
+            course_id='invalid',
+            list_type='per_course',
+            bookmarks_count=0,
+            page_size=10,
+            page_number=1
+        )
```

Comment:
Once the `course_id` is parsed in `paginate_queryset()` there should not be any events emitted here.